### PR TITLE
Minor cleanup to TSP queue table

### PR DESCRIPTION
**Pull request by chrisgilmerproj**
## Description

Cleanup the TSP queue table before the end of B&M:

- Fix GBL.
- Capitalize Status. 
- Update pickup date to best available. 
- Remove delivery date.

## Reviewer Notes

There aren't really good wireframes for this so its a best effort.  http://slicedbreadclient.com/dds/dp3/wireframes/TSP/round3/TSP%20NewShip%20Queue.png

## Setup

```sh
make db_dev_reset && make db_dev_migrate && go run ./cmd/generate_test_data/main.go -scenario=7
```

## Code Review Verification Steps

* [x] Request review from a member of a different team.
* [x] Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/161268264) for this change

## Screenshots

screen shot 2018-10-16 at 2 21 16 pm

### CODE REVIEW

**Comment by kahlouie** [comment]
Oh, on second thought, maybe you could add tests about the GBL id into e2e tests?

**Comment by chrisgilmerproj** [comment]
I'll definitely add a test, good catch.

**Comment by chrisgilmerproj** [comment]
I added a integration test for this.  If it passes through the circleci checks then I'll merge it.

File: cypress/integration/tsp/tspUser.js [hunk 1]
```diff
@@ -34,8 +34,10 @@ function tspUserViewsShipments() {
     .parentsUntil('div.rt-tr-group')
     .contains('div', 'Submitted, HHG')
     .parentsUntil('div.rt-tr-group')
-    .contains('div', 'US88 to Region 2')
-    .parentsUntil('div.rt-tr-group')
+    // TODO: cgilmer 2018/10/17 CircleCI seems to get this as 'US17 to Region 2'
+    // Should figure out why this is happening
```
[the remark this answers is not in the document]
Comment (chrisgilmerproj) [reply cypress/integration/tsp/tspUser.js]: Done! https://www.pivotaltracker.com/story/show/161322132

**Pull request merged**